draft-ietf-scitt-architecture.md

```diff
@@ -192,7 +192,7 @@ In COSE, Feed is a dedicated header attribute in the protected header of the Env
 Issuer:
 
 : an entity that creates Signed Statements about software Artifacts in the supply chain.
-An Issuer may be the owner or author of software Artifacts, or an independent third party such as a reviewer or an endorser.
+An Issuer may be the owner or author of Artifacts, or an independent third party such as a reviewer or an endorser.
```

Comment:
@JAG-UK I think software was deliberately added as the current scope was only `software`

Reply:
Sorry I just realized that software is already used in 194 so fine with it: now.